controllers/users/demoVerifyUser.js

```diff
@@ -25,6 +24,15 @@ module.exports = async (req, res) => {
           message: 'User has banned by admin'
         });
       }
+      if (userData.verified_status !== 'VERIFIED') {
+        return res.status(401).json({
+          code: 401,
+          data: false,
+          is_verified: true,
+          message: 'User is not verified'
+        });
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for checking the `verified_status` of a user is correct. However, it would be better to use constants for such string comparisons to avoid potential bugs due to typos and improve maintainability. Consider defining a constant for 'VERIFIED' at the top of your file or in a separate constants file.

```diff
- if (userData.verified_status !== 'VERIFIED') {
+ const VERIFIED_STATUS = 'VERIFIED';
+ if (userData.verified_status !== VERIFIED_STATUS) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->